src/site/stages/build/drupal/graphql/vamcPoliciesPage.graphql.js

```diff
@@ -0,0 +1,41 @@
+const policiesPageFragment = `
+  fragment policiesPageFragment on NodeVamcSystemPoliciesPage {
+    title
+    status
+    entityBundle
+    entityUrl {
+      path
+    }    
+    fieldVamcVisitationPolicy {
+      processed
+    }
+    fieldVamcOtherPolicies {
+      processed
+    }
+  }
+`;
+
+// TODO: add when content has been published:
+// query GetPolicyPages($onlyPublishedContent: Boolean!) {
+// { field: "status", value: ["1"], enabled: $onlyPublishedContent },
```

Comment:
If you're developing via the preview server, you could go ahead and add this `$onlyPublishedContent` piece

Reply:
Oh if you are developing using the `yarn build:content` then you have it right the way it is. But if you are doing `yarn preview` then navigating to `/preview?nodeId=1234` you cold go ahead and add the `onlyPublishedContent` piece bc the preview server doesn't care about published state.